### PR TITLE
Bug 1816724 - Show previous ETP message in onboarding if TCP is not enabled

### PR DESCRIPTION
**Pull request by Mugurell**
Restore showing a different description of the Tracking Protection feature in application's onboarding if the Total Cookie Protection feature is not yet public.
The new description mentioning Total Cookie Protection will only be shown if the Total Cookie Protection CFR is to also be shown.

| If TCP is enabled | If TCP is disabled |
| --- | --- |
|  |  |

### Pull Request checklist

- [x] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [x] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [x] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation
https://bugzilla.mozilla.org/show_bug.cgi?id=1816724

### CODE REVIEW

File: fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1308,6 +1308,8 @@
     <string name="onboarding_firefox_account_sync_is_on">Sync is on</string>
     <!-- Text for the tracking protection onboarding card header -->
     <string name="onboarding_tracking_protection_header">Privacy protection by default</string>
+    <!-- Text for the tracking protection card description. The first parameter is the name of the application.-->
+    <string name="onboarding_tracking_protection_description_old">%1$s automatically stops companies from secretly following you around the web.</string>
```
Comment (Mugurell) [inline fenix/app/src/main/res/values/strings.xml:1312]: Using the same string as before - https://github.com/mozilla-mobile/firefox-android/commit/b2845db6f9d56769a697dd062453f203e57ec36b?diff=unified#diff-b24d2928f9f4594421c030b62d3e5e070c9fd5600c871ff9a8c24a518fd02f96R1208

File: fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingTrackingProtectionViewHolder.kt [hunk 2]
```diff
@@ -18,19 +21,33 @@ class OnboardingTrackingProtectionViewHolder(view: View) : RecyclerView.ViewHold
...
+        val isTCPPublic = view.context.settings().enabledTotalCookieProtectionCFR
+        setupDescriptionText(view.context, isTCPPublic)
 
         val isTrackingProtectionEnabled = view.context.settings().shouldUseTrackingProtection
         setupRadioGroup(isTrackingProtectionEnabled)
         updateRadioGroupState(isTrackingProtectionEnabled)
     }
 
+    private fun setupDescriptionText(context: Context, shuldUseNewDescription: Boolean) {
+        if (!shuldUseNewDescription) {
```
Comment (Amejia481) [inline fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingTrackingProtectionViewHolder.kt:43]: nit: `UseNewDescription`

**Review by Amejia481 — APPROVED** [review]
Looks good!
There is CI failure, that we have to address :) 

```
[task 2023-02-15T16:59:37.215Z] > Task :ktlint
[task 2023-02-15T16:59:37.215Z] /builds/worker/checkouts/vcs/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingTrackingProtectionViewHolder.kt:10:1: Unused import (no-unused-imports)
[task 2023-02-15T16:59:37.215Z] /builds/worker/checkouts/vcs/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingTrackingProtectionViewHolder.kt:46:74: Argument should be on a separate line (unless all arguments can fit a single line) (argument-list-wrapping)
[task 2023-02-15T16:59:37.215Z] /builds/worker/checkouts/vcs/fenix/app/src/main/java/org/mozilla/fenix/home/sessioncontrol/viewholders/onboarding/OnboardingTrackingProtectionViewHolder.kt:46:81: Missing trailing comma before ")" (trailing-comma-on-call-site)
[task 2023-02-15T16:59:41.715Z] 
[task 2023-02-15T16:59:41.715Z] > Task :ktlint FAILED
[task 2023-02-15T16:59:41.715Z] 
[task 2023-02-15T16:59:41.715Z] FAILURE: Build failed with an exception.
```

**Comment by Mugurell** [comment]
@Mergifyio backport releases_v111

**Pull request merged**